src/base/static/components/app.js

```diff
@@ -68,14 +99,84 @@ const TemplateContainer = styled("div")(props => ({
 }));
 
 const Fallback = () => {
-  // TODO: render a spinner here...
-  return <div />;
+  return <Spinner />;
 };
 
 class App extends Component {
   templateContainerRef = createRef();
 
   async componentDidMount() {
```

Comment:
Duck loading and paginated Place loading now happens in `componentDidMount` of `App`.